internal/core/data/v2/application/event_test.go

```diff
@@ -178,3 +179,17 @@ func TestEventById(t *testing.T) {
 		})
 	}
 }
+
+func TestEventTotalCount(t *testing.T) {
+	dbClientMock := newMockDB(true)
+
+	dic := mocks.NewMockDIC()
+	dic.Update(di.ServiceConstructorMap{
+		v2DataContainer.DBClientInterfaceName: func(get di.Get) interface{} {
+			return dbClientMock
+		},
+	})
+
+	_, err := EventTotalCount(dic)
```

Comment:
Doesn't return an expected count that can be checked?

Reply:
modified